core/controllers/classifier.py

```diff
@@ -125,3 +125,31 @@ def post(self):
         classifier_services.mark_training_job_complete(job_id)
 
         return self.render_json({})
+
+
+class NextJobHandler(base.BaseHandler):
+    """This handler fetches next job to be processed according to the time
+    and sends back job_id, algorithm_id and training data to the VM.
+    """
+    REQUIRE_PAYLOAD_CSRF_CHECK = False
+
+    def post(self):
+        """Handles POST requests. """
+        signature = self.payload.get('signature')
+        vm_id = self.payload.get('vm_id')
+        message = json.dumps({})
```

Comment:
For simplicity and ease of understanding, let's keep the logic here exactly the same as in TrainedClassifierHandler (i.e. read the message sent). The VM will be sending a request with some sort of payload anyway (which in this case is expected to be empty).